Sources/NIOSSL/SSLCallbacks.swift

```diff
@@ -130,6 +130,22 @@ extension KeyLogCallbackManager {
     }
 }
 
+/// PSK Identity Callback Manager
+public typealias NIOPSKClientIdentityCallback = (NIOSSLSecureBytes?) -> (key: NIOSSLSecureBytes, identity: NIOSSLSecureBytes)
+public typealias NIOPSKServerIdentityCallback = (NIOSSLSecureBytes?, NIOSSLSecureBytes?) -> (key: NIOSSLSecureBytes, identity: NIOSSLSecureBytes)
```

Comment:
These callbacks need error reporting paths: they should be allowed to throw.

Reply:
Resolved in 516a89f.